src/ssl.c

```diff
@@ -49,6 +49,8 @@
     #include <errno.h>
 #endif
 
+static char currentCAName[ASN_NAME_MAX];
```

Comment:
These should be allocated and placed into the WOLFSSL object, not shared variable. This is not thread-safe.